frontend/src/bundles/common/components/loader/libs/constants/spin-animation.constant.ts

```diff
@@ -0,0 +1,8 @@
+import { keyframes } from '@chakra-ui/react';
```

Comment:
Create a wrapper for it so we can reuse it elsewhere in our app

Reply:
I think we can keep it like this, because it is just helper function